animation_nodes/data_structures/meshes/mesh_data.pyx

```diff
@@ -60,11 +61,13 @@ cdef class Mesh:
         self.derivedMeshDataCache = {}
         self.uvMaps = OrderedDict()
         self.vertexColorLayers = OrderedDict()
+        self.attributes = OrderedDict()
 
     def getMeshProperties(self):
         return (
-            (self.uvMaps, Vector2DList),
-            (self.vertexColorLayers, ColorList),
+            (self.uvMaps, Attribute),
+            (self.vertexColorLayers, Attribute),
+            (self.attributes, Attribute),
         )
```

Comment:
This should be removed. You can loop over `attributes` directly.

Reply:
Done.